commands/help.rb

```diff
@@ -101,16 +101,16 @@ def self.help(command)
       puts <<~EOT
         Look for package(s).
         Usage: crew search [-v|--verbose] [<pattern> ...]
-        If <pattern> is omitted, all packages will be returned.
+        Both the name and description of packages will be searched.
```

Comment:
The only way anything is searched is if there is a pattern to filter the results.

Reply:
Yes? I'm not really sure what you're commenting on here.